src/test/phold/phold-parallel.yaml

```diff
@@ -0,0 +1 @@
+phold.yaml
```

Comment:
Huh; I guess this is an 'include'? I didn't know yaml supported that; [SO](https://stackoverflow.com/questions/528281/how-can-i-include-a-yaml-file-inside-another) seems to think it doesn't :). Since this isn't standard yaml could probably use a comment

Reply:
It's a symbolic link checked into the repo.